src/features/filterEventsByCity.test.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { mount } from 'enzyme';
```

Comment:
This line is duplicating the import of `mount`. You already defined in line 6. Kindly delete it